egs/jsalt18e2e/asr1/run.sh

```diff
@@ -168,20 +168,19 @@ if [ ${stage} -le 0 ]; then
     done
 fi
 
-feat_tr_dir=${dumpdir}/${train_set}/delta${do_delta}; mkdir -p ${feat_tr_dir}
-feat_dt_dir=${dumpdir}/${train_dev}/delta${do_delta}; mkdir -p ${feat_dt_dir}
+feat_tr_dir=${dumpdir}/${train_set}/delta${do_delta}; mkdir -p ${feat_tr_dir}/json_${train_set}
+feat_dt_dir=${dumpdir}/${train_dev}/delta${do_delta}; mkdir -p ${feat_dt_dir}/json_${train_set}
```

Comment:
For different subset experiment, the generated data.json is different for all train/test datasets due to the different lang dir.

Reply:
We don't need lang dir.
Also is `/json_${train_set}` required?

My concern is that with subset, we actually need to prepare the dev and test set directories depending on the ${train_set} as it depend on CMN obtained by ${train_set}. 
How about 
`feat_dt_dir=${dumpdir}/${train_dev}_${train_set}/delta${do_delta}`
`feat_recog_dir=${dumpdir}/${rtask}_${train_set}/delta${do_delta}`

Reply:
We use only one language dictionary trained from the whole set other than forming language dictionary from subset?

Reply:
Yes, and I think we may not have to make it depending on ${train_set}, as we always use the language dictionary trained from the whole set.

Reply:
Gotcha, that makes sense. Please find the new commit for the update.